lib/iaas/BaseCloudClient.js

```diff
@@ -12,7 +12,7 @@ const CloudProviderError = {
     return err.statusCode === 401 || err.failCode === 'Unauthorized';
   },
   Forbidden: err => {
-    return err.statusCode === 403 || err.code === 404;
+    return err.statusCode === 403 || err.code === 403;
```

Comment:
use status code value from constant. We are gradually improving on this in our whole code base. If the PR needs to be merged immediately, then not required to fix it now.

Reply:
Sure, will do.